src/test/java/com/hocs/test/glue/CreateCaseStepDefs.java

```diff
@@ -45,6 +49,30 @@ public void iAmPresentedWith(String userView) {
         }
     }
 
+    
+    @Given("^I create a single case \"([^\"]*)\"$")
+    public void iCreateACaseTypeSpecificCase(String caseType) {
+
+        switch (caseType.toUpperCase()) {
+            case "DCU MIN":
+                createCase.createDCUMinSingleCase();
+                break;
+            case "DCU N10":
+                createCase.createDC10SingleCase();
+                break;
+            case "DCU TRO":
+                createCase.createDCTROSingleCase();
+                break;
+            default:
+                System.out.println(createCase
```

Comment:
this should be caseType not createCase

System.out.println(createCase
```suggestion
System.out.println(caseType
```